src/components/forms/checkbox.jsx

```diff
@@ -15,9 +15,7 @@ var Checkbox = React.createClass({
             this.props.className
         );
         return (
-            <div className={classes}>
-                <FRCCheckbox {... this.props} />
-            </div>
+            <FRCCheckbox elementWrapperClassName={classes} {... this.props} />
```

Comment:
I did it this way so that the stylings could be uniformly applied to elements inside of `.form-group` elements. The way FRC works, it builds a `Row` around everything, so I was seeing this more the outermost wrapper around the element itself, even if it wasn't outside of `Row`. It made more sense for me for it to be `.form-group > .[element]` rather than `.[element] > .form-group`.

The other reason is because this is the only place we do this – I added that wrapper in the initial commit of this feature because applying `className` directly onto `FRCCheckbox` wasn't working. We had this set up, applying `className` to FRC elements to provide custom formatting, everywhere. However, I discovered in this bug that FRC was basically ignoring all of those classnames, and not applying them, since that's not how it works. 

When I learned how `elementWrapperClassName` worked, it seemed to make more sense to me to use that, rather than adding another `div` wrapper around all FRC components to make this uniform.